src/plugins/Debug.ts

```diff
@@ -24,6 +28,18 @@ class Debug extends GenericPlugin {
     const { container } = this.ad;
 
     this.debugOverlay = insertElement('div', { style: OVERLAY_STYLE }, container);
+    googletag.cmd.push(() => {
```

Comment:
I decided to add a second event listener because the alternative would be to attach metadata to each ad regardless of the plugin being used in the network. This is obviously not ideal because we only need the data if the plugin is used and this plugin is not used in production.

Reply:
What if we hook into the ad event bus which should pass the event back?

this.ad.on('render', (event) => {
});